src/applications/proxy-rewrite/proxy-rewrite-whitelist.json

```diff
@@ -0,0 +1,29 @@
+{
+    "proxyRewriteWhitelist": [
+    {
+        "hostname": "dev.va.gov",
+        "pathnameBeginning": "/health/",
+        "cookieOnly": false
+    },
+    {
+        "hostname": "staging.va.gov",
+        "pathnameBeginning": "/health/",
+        "cookieOnly": false
+    },
+    {
+        "hostname": "preview.va.gov",
+        "pathnameBeginning": "/health/",
```

Comment:
this fulfills the requirements for now- will probably move away from json and store regex in code if we need to accommodate more complicated patterns.